README.md

```diff
@@ -443,6 +443,8 @@ An advanced agenda component that can display interactive listings for calendar
     '2012-05-17': {marked: true},
     '2012-05-18': {disabled: true}
   }}
+  // If disabledByDefaut={true} dates flaged as not disabled will be enabled. Default = false
+  disableByDefault={true}
```

Comment:
You have a typo. It should be 'disabledByDefault' (you're missing a 'd' in the end of 'disable')